vmm/src/cpu.rs

```diff
@@ -919,12 +921,14 @@ impl CpuManager {
 
                                 #[cfg(feature = "kvm")]
                                 {
-                                    vcpu.lock().as_ref().unwrap().vcpu.set_immediate_exit(true);
-                                    if !matches!(vcpu.lock().unwrap().run(), Ok(VmExit::Ignore)) {
-                                        error!("Unexpected VM exit on \"immediate_exit\" run");
-                                        break;
+                                    if matches!(hypervisor_type, HypervisorType::Kvm) {
```

Comment:
@russell-islam My main point was you can move `#[cfg(feature = "kvm")]` right before the `if matches!(hypervisor_type, HypervisorType::Kvm)` statement, which would avoid the extra parentheses and make the patch smaller/cleaner (4 lines of changed in total).

Reply:
@likebreath Sorry, I missed your comment since there was an already overlapping comment.  You are right a layer of indentation/grouping could be removed. Could you create a follow-on PR?

Reply:
I will create a follow-up. Sorry for the delay!!